### PR TITLE
Add google dictionary API.

### PR DESCRIPTION
**Pull request by meetDeveloper**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by meetDeveloper** [comment]
Also the dictionary entry I added is open source, source code can be found here https://github.com/meetDeveloper/googleDictionaryAPI.

File: README.md [hunk 1]
```diff
@@ -296,6 +296,7 @@ API | Description | Auth | HTTPS | CORS |
 ### Dictionaries
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
+| [Google Dictionary API](https://dictionaryapi.dev) | Word definitions, examples, pronunciations, synonyms, and others | No | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:299]: Can you change the title `Google Dictionary API` to `Google Dictionary`, please?

```suggestion
| [Google Dictionary](https://dictionaryapi.dev) | Word definitions, examples, pronunciations, synonyms, and others | No | Yes | Yes |
```

Title should not end with "... API". Every entry is an API here! 😅

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for contribution @meetDeveloper 😄

**Comment by marekdano** [comment]
@meetDeveloper can you please check the suggestion from @matheusfelipeog and update your record accordingly?

**Comment by ElWeroRanchero** [comment]
Git//ElWeroRanchero

**Comment by meetDeveloper** [comment]
@ElWeroRanchero  This has already been added, closing this PR.

**Pull request closed without merging**